### ISSUE TITLE
1050 ti 4gb

### ISSUE THREAD

**Issue by Green929** [trexminer/T-Rex#579]
is there a way you can mine eth with a 4gb card?

**Comment by Crisis83** [comment]
No, the DAG is too large to fit in GPU memory. Even with a 6GB card there isn't much left on Windows miners.
